notebooks/05_operationalize/pipeline-demo/aml_config/library.runconfig

```diff
@@ -0,0 +1,21 @@
+environment:

```

Comment:
is this something that is automatically created? do we need it here?

Reply:
Library.runconfig is automatically created by AML pipeline script, which is needed to run script. It is the list of required library. I feel it would be good to have it in the repo so others can check whether their created configfile is right or not.

Reply:
Ah I see. In that case it might be easier to create a README in the pipeline-demo folder describing what each file is / how it's used and what the expected output from autogenerated files is. That way you can show the contents of the runconfig and other files without needed to check in code that could collide with user generated files.